WordPress/wordpress-develop
# Add images for block editor's Welcome Guide

Trac ticket: TBD 

Adds the image assets necessary for https://github.com/WordPress/gutenberg/pull/32026.

To test:

1. Clone the `wordpress-develop` repo, check out https://github.com/WordPress/wordpress-develop/pull/1269, build using `npm i; grunt build`.
2. Clone the `gutenberg` repo, check out https://github.com/WordPress/gutenberg/pull/32026, build using `npm i; npm run build`.
3. Configure your environment to run both local copies. For example, if using `wp-env`, modify `gutenberg/.wp-env.json` to set `"core": "/path/to/wordpress-develop/build"`, then run `wp-env start`.
4. Go to Posts → Add New, open the Options menu, and click Welcome Guide. The guide's images should appear normally.
5. Go to Appearance → Widgets, open the Options menu, and click Welcome Guide. The guide's images should appear normally.
6. Go to Appearance → Customize → Widgets, open the Options menu, and click Welcome Guide. The guide's image should appear normally.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**
